watchtower-plugin/src/net/http.rs

```diff
@@ -339,9 +426,13 @@ mod tests {
         let (tower_sk, tower_pk) = cryptography::get_random_keypair();
         let appointment = generate_random_appointment(None);
 
+        #[cfg(feature = "accountable")]
         let appointment_receipt = get_random_appointment_receipt(tower_sk);
+        #[cfg(feature = "accountable")]
         let add_appointment_response =
             get_dummy_add_appointment_response(appointment.locator, &appointment_receipt);
+        #[cfg(not(feature = "accountable"))]
```

Comment:
This can be simplified, do a single assignment with a conditional call

Reply:
done